google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/it/UnaryMetricsMetadataIT.java

```diff
@@ -69,29 +70,36 @@ public void testSuccess() throws Exception {
     List<Cluster> clusters = clustersFuture.get(1, TimeUnit.MINUTES);
 
     // give opencensus some time to populate view data
-    Thread.sleep(100);
+    Thread.sleep(1000);
 
     List<String> tagValueStrings = StatsWrapper.getOperationLatencyViewTagValueStrings();
     assertThat(tagValueStrings).contains(clusters.get(0).getZone());
     assertThat(tagValueStrings).contains(clusters.get(0).getId());
```

Comment:
I dont love blindly extending sleeps. Can we switch this to polling instead?

for(int i=0; i<10;i++) {
   if(StatsWrapper.getOperationLatencyViewTagValueStrings()).contains(clusters.get(0).getId()) {
    break;
  }

    List<String> tagValueStrings = StatsWrapper.getOperationLatencyViewTagValueStrings();
    assertThat(tagValueStrings).contains(clusters.get(0).getZone());
    assertThat(tagValueStrings).contains(clusters.get(0).getId());
}


same for below

Reply:
updated!